### DISCUSSION TITLE
Adobe Animate and iOS app for M1 compatibility

### DISCUSSION THREAD

**Discussion by elemdeso** [airsdk/Adobe-Runtime-Support#2004] [Q&A]
Can I build an iOS app on an Intel Mac using Adobe Animate and get it to run on an M1 Mac? If not, if I build it on an M1 Mac, will it be playable - bearing in mind Animate is not Arm native yet (depends on Rosetta)?
Thanks.

**Comment by ajwfrost** [comment]
Yes, if you build an iOS app on whichever type of mac (or Windows) computer, it's possible to run it on an M1 mac
https://developer.apple.com/documentation/apple-silicon/running-your-ios-apps-on-macos

The fact that Animate is only compiled to x86_64 doesn't make a difference on the AIR-based applications, and whichever type of Mac you are running this on, you'll always get arm64 output in an IPA file or x86_64 and arm64 (universal binary) if you target a Mac bundle.

thanks

**Reply by elemdeso** [reply]
Thank you, Andrew. Courteous and clear - as ever.
